README.md

```diff
@@ -70,12 +70,14 @@ jobs:
 
 ```yml
 name: GitOps
-on: [push]
+on: [ push ]
 
 jobs:
   gitops:
     uses: Staffbase/gha-workflows/.github/workflows/template_gitops.yml@v1.3.1
     with:
+      # optional: build and push the Docker image, default: true
+      dockerenabled: true or false
```

Comment:
I would prefer this for the reason that if you copy it and really want to change it, then it has to be false, otherwise you delete it again since the default value is true.

```suggestion
      dockerenabled: false
```